### PR TITLE
Summon Artifacts

### PR DESCRIPTION
**Pull request by gurfan**
Costs 40 points. Fun for the whole crew.

Not included in the summon: spellbook/potion bundles, butterfly knife (doesnt work because of faction nonsense), other summon spells (oh god)

:cl:
 * rscadd: Wizards can now summon artifacts for the crew.

### CODE REVIEW

**Comment by gurfan** [comment]
stay tuned for more summon gunk spells coming to a pr near you

**Comment by BomberBro** [comment]
am I reading this wrong or can it give the crew staffs of change? if so, it basically signals the death knell of the round when used

**Comment by gurfan** [comment]
> am I reading this wrong or can it give the crew staffs of change? if so, it basically signals the death knell of the round when used

Why of course it can. If staffs of change will be hated that much i can remove them from the list

**Comment by BomberBro** [comment]
too much gunk potential then, we shitcanned barry for making summon bombs and this is worse

**Comment by gurfan** [comment]
> too much gunk potential then, we shitcanned barry for making summon bombs and this is worse

fine, its off the summon list

File: code/game/gamemodes/wizard/rightandwrong.dm [hunk 1]
```diff
@@ -416,3 +420,105 @@
...
+			new /obj/item/weapon/tank/emergency_oxygen/double/wizard(get_turf(src))
+		if("staffanimate")
+			new /obj/item/weapon/gun/energy/staff/animate(get_turf(src))
+		if("staffnecro")
+			new /obj/item/weapon/gun/energy/staff/necro(get_turf(src))
+		if("apprentice") //lol
+			new /obj/item/wizard_apprentice_contract(get_turf(src))
+		if("scyingorb")
+			new /obj/item/weapon/scrying(get_turf(src))
+			mutations.Add(M_XRAY)
```
Comment (DamianX) [inline code/game/gamemodes/wizard/rightandwrong.dm:512]: this is the effect that holding the scrying orb gives you, why is this here?

Comment (gurfan) [reply code/game/gamemodes/wizard/rightandwrong.dm:512]: the scying orb doesn't do anything when held, it lets you ghost when you use it. It gives a wizard x-ray vision when they purchase it, so I'm giving the same perk to people who recieve it

Comment (DamianX) [reply code/game/gamemodes/wizard/rightandwrong.dm:512]: Huh, I remembered wrong
